src/pages/integrations/shapeshift/shapeshift-confirm/shapeshift-confirm.ts

```diff
@@ -442,17 +447,25 @@ export class ShapeshiftConfirmPage {
         txp.feeLevel = this.configWallet.settings.feeLevel || 'normal';
       }
 
-      this.walletProvider
-        .createTx(wallet, txp)
-        .then(ctxp => {
-          return resolve(ctxp);
+      return this.walletProvider
+        .getAddress(wallet, false)
```

Comment:
Why it's necessary to create a new address?

Reply:
For txp.from property. Can remove this change until shape shift is ready for ETH

Reply:
Also it does not create a new address because the second param is `false` in `.getAddress(wallet, false)`

Reply:
Reverted back to master